### PR TITLE
argparse, ipaddr, python-ldap, python-openid, simplejson

### PR DESCRIPTION
**Pull request by cclauss**
I might not have this quite right so please feel free to reject.

I have [a similar file](https://github.com/cclauss/python3wos_asyncio/blob/master/static_parent_dir/equivalent_modules.json) but not so nearly complete for https://github.com/cclauss/python3wos_asyncio where we are trying to determine which of the top PyPI packages are missing the needed PyPI trove classifiers and which have `equivalents` which is similar but not exactly the same as your `overrides`.

One of my goals is to _push package maintainers to update their trove classifiers_ to make it clear that their package supportes Python3.

### CODE REVIEW

File: caniusepython3/overrides.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+    "argparse": "https://docs.python.org/3/library/argparse.html",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]: Already lists Python 3 support in their trove classifier.

File: caniusepython3/overrides.json [hunk 2]
```diff
@@ -35,12 +37,15 @@
...
     "pysqlite": "http://docs.python.org/3/library/sqlite3.html",
+    "python-ldap": "https://pypi.python.org/pypi/pyldap",
+    "python-openid": "https://github.com/necaris/python3-openid",
     "pyvirtualdisplay": "https://pypi.python.org/pypi/PyVirtualDisplay",
     "regex": "https://pypi.python.org/pypi/regex",
     "requests-oauthlib": "https://pypi.python.org/pypi/requests-oauthlib",
     "ssh": "https://github.com/bitprophet/ssh/",
     "ssl": "http://docs.python.org/3/library/ssl.html",
     "suds": "https://pypi.python.org/pypi/suds-jurko",
+    "simplejson": "https://docs.python.org/3/library/json.html",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]: Already lists Python 3 support in their trove classifier.

File: caniusepython3/overrides.json [hunk 3]
```diff
@@ -35,12 +37,15 @@
     "pypdf": "https://pypi.python.org/pypi/PyPDF2",
     "pyrss2gen": "https://pypi.python.org/pypi/PyRSS2Gen",
     "pysqlite": "http://docs.python.org/3/library/sqlite3.html",
+    "python-ldap": "https://pypi.python.org/pypi/pyldap",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]: PyPI page for pyldap doesn't mention Python 3 support.

**Comment by brettcannon** [comment]
So the stdlib overrides that declare Python 3 support in their PyPI equivalents don't need to be listed. I also left other comments on some projects where your link doesn't provide the exact evidence for Python 3 support.

File: caniusepython3/overrides.json (hunk 3 above)
Comment (cclauss) [inline caniusepython3/overrides.json]: Agreed.  The trove classifiers are in [pyldap's setup.py file](https://github.com/pyldap/pyldap/blob/master/setup.py#L120) but not in the PyPI record.  See https://github.com/pyldap/pyldap/pull/52

Comment (brettcannon) [inline caniusepython3/overrides.json]: Then please update  the URL to point to the setup.py file.

**Comment by brettcannon** [comment]
Just to double-check about pyldap since I see you authored the change adding the classifier, @cclauss , the support is in the publicly released code on PyPI, correct?

**Comment by brettcannon** [comment]
It's been over a month since I asked for clarifications for something, so I'm closing this. If you come back to this, @cclauss , just leave a comment and answer my last question.

**Pull request closed without merging**